### PR TITLE
optimise docs

### PR DESCRIPTION
**Pull request by panyufeng920**
**\- What I did**

**\- How I did it**

**\- How to verify it**

**\- A picture of a cute animal (not mandatory but encouraged)**

Signed-off-by: 搏通 yufeng.pyf@alibaba-inc.com

### CODE REVIEW

File: docs/userguide/networking/dockernetworks.md [hunk 1]
```diff
@@ -138,7 +138,7 @@ $ docker run -itd --name=container2 busybox
 94447ca479852d29aeddca75c28f7104df3c3196d7b6d83061879e339946805c
 ```
 
-Inspecting the `bridge` network again after starting two containers shows both newly launched containers in the network. Their ids show up in the container
+Inspecting the `bridge` network again after starting two containers shows both newly launched containers in the network. Their ids show up in the "Containers".
```
Comment (thaJeztah) [inline docs/userguide/networking/dockernetworks.md]: Perhaps we should be more explicit, something like:

```Markdown
Their ids show up in the "Containers" section of `docker network inspect`:
```

wdyt?

**Comment by panyufeng920** [comment]
@thaJeztah Thank you, I have commited as your suggesting, pls review it.

**Comment by thaJeztah** [comment]
LGTM, thanks!

**Comment by calavera** [comment]
LGTM

**Pull request merged**